functions/nodejs-apps.bash

```diff
@@ -35,13 +35,15 @@ frontail_setup() {
   cond_redirect npm update -g frontail
   #
   frontail_base="/usr/lib/node_modules/frontail"
+  mkdir -p ${frontail_base}/preset ${frontail_base}/web/assets/styles
```

Comment:
Markus do you think that this will have any implications on this PR: https://github.com/openhab/openhabian/pull/601 ?

Reply:
sorry maybe because it's late but I don't see any linkage there ?

Reply:
Okey, nice